src/game.ts

```diff
@@ -1,51 +1,97 @@
-    public player: Player
-    private gameTexture: PIXI.Texture
-    public mail: MailScreen
-    private officeAssets: PIXI.Texture
-    private mailAssets: PIXI.Texture[];
+  public pixi: PIXI.Application;
+  private loader: AssetLoader;
+  public player: Player;
+  private gameTexture: PIXI.Texture;
+  public mail: MailScreen;
+  private officeAssets: PIXI.Texture;
+  private mailAssets: PIXI.Texture[];
+
+  //water parameters related
+  public waterParameters: WaterParam[];
+  private waterParamA: WaterParam;
+  private waterParamB: WaterParam;
+  private waterParamC: WaterParam;
+  // public waterModel: WaterModel;
+
+  constructor() {
+    PIXI.settings.ROUND_PIXELS = true;
+
+    // init game
+    this.pixi = new PIXI.Application({
+      autoDensity: true,
+      resolution: window.devicePixelRatio,
+    });
 
-    constructor() {
-        PIXI.settings.ROUND_PIXELS = true
+    document.body.appendChild(this.pixi.view as HTMLCanvasElement);
+    // Load images
+    this.loader = new AssetLoader(this);
+  }
 
-        // init game
-        this.pixi = new PIXI.Application({ autoDensity: true, resolution: window.devicePixelRatio })
+  loadCompleted() {
+    console.log("Load completed");
+    console.log(this.loader.textures);
 
-        document.body.appendChild(this.pixi.view as HTMLCanvasElement)
-        // Load images
-        this.loader = new AssetLoader(this)
-    }
+    this.gameTexture = this.loader.textures.Player["flowerTop"];
+    this.officeAssets = this.loader.textures.Office;
+    this.mailAssets = this.loader.textures.MailScreen;
 
-    async loadCompleted() {
-        console.log("Load completed")
-        console.log(this.loader.textures)
+    // this.player = new Player(this.gameTexture)
+    // this.pixi.stage.addChild(this.player)
 
-        this.gameTexture = this.loader.textures.Player['flowerTop']
-        this.officeAssets = this.loader.textures.Office
-        this.mailAssets = this.loader.textures.MailScreen
+    this.mail = new MailScreen(this.mailAssets, this);
+    this.pixi.stage.addChild(this.mail);
 
-        // this.player = new Player(this.gameTexture)
-        // this.pixi.stage.addChild(this.player)
+    this.mail.add(
+      "Lob lob lob",
+      "De zomer is aantocht het beloofd een warme en droge zomer te worden. Ons doel is om onze inwoners schoon en veilig zwemwater te kunnen bieden. Zodat zij het hoofd koel kunnen houden! \n\nJouw doel voor de komende week is; de waterkwaliteit verbeteren.",
+      0,
+      true,
+      "lob"
+    );
+    this.mail.add(
+      "Mail 1",
+      "This is the first maiwadawdawdwad wdmwaidmwa idmawid dadwad wl.",
+      0,
+      false,
+      "lob"
+    );
+    this.mail.add("Mail 3", "This is the third mail.", 0, false, "lob");
+    this.mail.add("Mail 4", "This is the third mail.", 0);
 
-        this.mail = new MailScreen(this.mailAssets, this)
-        this.pixi.stage.addChild(this.mail)
+    // Parameter initialization
+    this.waterParameters = [];
```

Comment:
netter als die geinit wordt in de contructor

Reply:
Ja word vervangen door een eigen class, die ook het tonen van de params (balkjes) doet etc

Reply:
alle params initializeren nu in de constructor. 
waterparameters[] weggehaald.